lib/transcryptor/instance.rb

```diff
@@ -45,9 +49,11 @@ def re_encrypt(table_name, attribute_name, old_opts, new_opts, other_columns = [
     encryptor_class = attr_encrypted_poro_class(attribute_name, new_opts, all_columns)
 
     rows.each do |old_row|
+      prehook.call(attribute_name, old_row, decryptor_class, old_opts)
```

Comment:
I'd like to standardise parameters in proc for `prehook` & `posthook`. Let's say:
```
prehook.call(old_row, decryptor_class)
prehook.call(new_row, encryptor_class)
```
or sth similar, because `table_name, attribute_name, old_opts, new_opts, re_encrypt_opts` is already known as input parameters of `re_encrypt_column` methods no need to pass them as proc params.